R/test.R

```diff
@@ -7,7 +7,7 @@ request_test <- function(template = "/get", ...) {
   }
 
   req <- request(the$test_app$url())
-  req <- req_template(req, template, ..., .env = caller_env())
+  req <- req_template(req, sub("^/", "", template), ..., .env = caller_env())
```

Comment:
This doesn't seem right to me — should this logic belong in `req_url_path_append()`?

I think I'd expect:

```
req1 <- request("http://test.com/x") %>% req_template("PATCH /y")
req2 <- request("http://test.com/x/") %>% req_template("PATCH /y")
expect_equal(req1$url, req2$url)
```

Reply:
That's what I would expect! But then I noticed in `test-req-url.R` (line 19):

```
req2 <- request("http://example.com/")

# specifically requested to add in both
  expect_equal(req_url_path_append(req2, "/index.html")$url, "http://example.com//index.html")
```

I don't know why but it is on purpose. So I left it as it was.

Reply:
I think we should say that `req_url_path_append()` will never make a `//` — if you do need it for some reason, you can use `req_url_path()`.

Reply:
Ok. What's the best way to handle this - a new PR?